client/views/actions/nick.tpl

```diff
@@ -1,3 +1,3 @@
 {{> ../user_name nick=from}}
-is now known as
+{{translate 'client.now_known_as'}}
 {{> ../user_name nick=new_nick}}
```

Comment:
Previous comment applies here as well, and most templates I believe. Could this be made so that the strings in the JSON files contain the other elements, like `{{from}} is now known as {{new_nick}}`?